- Fix up default paths
- Support env CC
- Don't build/install libtz.a
- Don't install man-pages provided by sys-apps/man-pages
- Move zic, zdump to sbin and tzselect to bin ala glibc
- Install posix zoneinfo into zoneinfo/posix/ instead of zoneinfo-posix/ ala glibc
- Install leaps zoneinfo into zoneinfo/right/ ala glibc

--- Makefile.makefile	2013-10-25 20:34:35.000000000 -0200
+++ Makefile	2013-10-28 17:22:11.697366378 -0200
@@ -45,21 +45,21 @@ POSIXRULES=	America/New_York
 
 # Everything gets put in subdirectories of. . .
 
-TOPDIR=		/usr/local
+TOPDIR=		/usr
 
 # "Compiled" time zone information is placed in the "TZDIR" directory
 # (and subdirectories).
 # Use an absolute path name for TZDIR unless you're just testing the software.
 
 TZDIR_BASENAME=	zoneinfo
-TZDIR=		$(TOPDIR)/etc/$(TZDIR_BASENAME)
+TZDIR=		$(TOPDIR)/share/$(TZDIR_BASENAME)
 
 # Types to try, as an alternative to time_t.  int64_t should be first.
 TIME_T_ALTERNATIVES= int64_t int32_t uint32_t uint64_t
 
 # The "tzselect", "zic", and "zdump" commands get installed in. . .
 
-ETCDIR=		$(TOPDIR)/etc
+SBINDIR=		$(TOPDIR)/sbin
 
 # If you "make INSTALL", the "date" command gets installed in. . .
 
@@ -298,8 +298,10 @@ GZIPFLAGS=	-9n
 
 ###############################################################################
 
-cc=		cc
-CC=		$(cc) -DTZDIR=\"$(TZDIR)\"
+CC+=		-DTZDIR=\"$(TZDIR)\"
+ifeq ($(NLS),1)
+CC += -DHAVE_GETTEXT=1 -DTZ_DOMAIN=\"libc\"
+endif
 
 AR=		ar
 
@@ -345,24 +347,22 @@ ENCHILADA=	$(COMMON) $(DOCS) $(SOURCES)
 
 SHELL=		/bin/sh
 
-all:		tzselect zic zdump libtz.a $(TABDATA)
+all:		tzselect zic zdump $(TABDATA)
 
 ALL:		all date
 
 install:	all $(DATA) $(REDO) $(MANS)
-		mkdir -p $(DESTDIR)$(ETCDIR) $(DESTDIR)$(TZDIR) \
-			$(DESTDIR)$(LIBDIR) \
+		mkdir -p $(DESTDIR)$(SBINDIR) $(DESTDIR)$(TZDIR) \
 			$(DESTDIR)$(MANDIR)/man3 $(DESTDIR)$(MANDIR)/man5 \
 			$(DESTDIR)$(MANDIR)/man8
 		$(ZIC) -y $(YEARISTYPE) \
 			-d $(DESTDIR)$(TZDIR) -l $(LOCALTIME) -p $(POSIXRULES)
-		cp -f iso3166.tab zone.tab $(DESTDIR)$(TZDIR)/.
-		cp tzselect zic zdump $(DESTDIR)$(ETCDIR)/.
-		cp libtz.a $(DESTDIR)$(LIBDIR)/.
-		$(RANLIB) $(DESTDIR)$(LIBDIR)/libtz.a
+		cp -f iso3166.tab zone.tab $(DESTDIR)$(TZDIR)/
+		-mkdir -p $(DESTDIR)$(TOPDIR) $(DESTDIR)$(SBINDIR)
+		cp zic zdump $(DESTDIR)$(SBINDIR)/.
+		-mkdir -p $(DESTDIR)$(TOPDIR) $(DESTDIR)$(BINDIR)
+		cp tzselect $(DESTDIR)$(BINDIR)/.
 		cp -f newctime.3 newtzset.3 $(DESTDIR)$(MANDIR)/man3/.
-		cp -f tzfile.5 $(DESTDIR)$(MANDIR)/man5/.
-		cp -f tzselect.8 zdump.8 zic.8 $(DESTDIR)$(MANDIR)/man8/.
 
 INSTALL:	ALL install date.1
 		mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1
@@ -405,19 +405,19 @@ right_only:	zic leapseconds $(TDATA)
 # You must replace all of $(TZDIR) to switch from not using leap seconds
 # to using them, or vice versa.
 right_posix:	right_only leapseconds
-		rm -fr $(DESTDIR)$(TZDIR)-leaps
-		ln -s $(TZDIR_BASENAME) $(DESTDIR)$(TZDIR)-leaps || \
-		  $(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)-leaps \
+		rm -fr $(DESTDIR)$(TZDIR)/right
+		ln -s ../$(TZDIR_BASENAME) $(DESTDIR)$(TZDIR)/right || \
+		  $(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)/right \
 			-L leapseconds $(TDATA)
-		$(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)-posix \
+		$(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)/posix \
 			-L /dev/null $(TDATA)
 
 posix_right:	posix_only leapseconds
-		rm -fr $(DESTDIR)$(TZDIR)-posix
-		ln -s $(TZDIR_BASENAME) $(DESTDIR)$(TZDIR)-posix || \
-		  $(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)-posix \
+		rm -fr $(DESTDIR)$(TZDIR)/posix
+		ln -s ../$(TZDIR_BASENAME) $(DESTDIR)$(TZDIR)/posix || \
+		  $(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)/posix \
 			-L /dev/null $(TDATA)
-		$(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)-leaps \
+		$(ZIC) -y $(YEARISTYPE) -d $(DESTDIR)$(TZDIR)/right \
 			-L leapseconds $(TDATA)
 
 zones:		$(REDO)
@@ -440,7 +440,7 @@ tzselect:	tzselect.ksh
 			<$? >$@
 		chmod +x $@
 
-check:		check_character_set check_tables check_web
+check:		check_character_set check_tables
 
 check_character_set: $(ENCHILADA)
 		sharp='#'; ! grep -n $(INVALID_CHAR) $(ENCHILADA)
